enterprise/frontend/src/metabase-enterprise/whitelabel/components/LoginPageIllustrationDescription.js

```diff
@@ -0,0 +1,26 @@
+import { t } from "ttag";
+
+import { Flex, Icon, Stack, Text, HoverCard } from "metabase/ui";
+
+export function LoginPageIllustrationDescription() {
+  return (
+    <Text fw="bold" transform="none">
+      <Flex align="center">
+        {t`Login page`}
+        <HoverCard position="top-start">
+          <HoverCard.Target>
+            <Icon aria-hidden name="info" />
```

Comment:
`aria-hidden` makes sure we could use `*ByLabelText("Login page")` later on, though I'm not using that right now.